star_tides/services/databases/mongo/models/location_model.py

```diff
@@ -0,0 +1,13 @@
+'''Objects and utilities for working with geographic locations at the database
+level.
+'''
+
+from mongoengine import EmbeddedDocument, IntField, StringField
+
+class LocationModel(EmbeddedDocument):
+    '''Location model for attaching geographic information to a model via a
+    mongoengine.EmbeddedDocumentField.
+    '''
+
+    iso_3166_1_country_code = IntField(required=True)
+    arbitrary_text = StringField(required=False)
```

Comment:
What goes in here?

Reply:
Arbitrary text! It may be a country, city, neighborhood, whatever. It provides additional description on top of the country code. See https://docs.google.com/document/d/1ssodW1nXXPWGl0rFdbBFaq1mWCSCh2UoLinGZjeY8SE/edit?resourcekey=0-xpyo93lihGxVenywxlJQ9A#heading=h.rk2ebu4quj8y for more.

We're going with this instead of the usual Street, City, ZIP, etc. since not all locations may map to that exactly.